prisma/schema.prisma

```diff
@@ -55,21 +50,70 @@ model Session {
     user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
 }
 
-model User {
-    id            String    @id @default(cuid())
-    name          String?
-    email         String?   @unique
-    emailVerified DateTime?
-    image         String?
-    accounts      Account[]
-    sessions      Session[]
-    posts         Post[]
-}
-
 model VerificationToken {
     identifier String
     token      String   @unique
     expires    DateTime
 
     @@unique([identifier, token])
 }
+
+
+model Income {
+    id          Int      @id @default(autoincrement())
+    amount      Float
+    date        DateTime
+    description String?
+    source      String
+    userId      String
+    user        User     @relation(fields: [userId], references: [id])
+
+    @@index([source])
+}
+
+model Expense {
+    id          Int      @id @default(autoincrement())
+    amount      Float
+    date        DateTime
+    category    String
```

Comment:
It makes sense to have these categories as an enum unless you feel otherwise.
https://github.com/owenmccomas/fancy-financey/issues/2